Header header
float32 lat
float32 lon
float32 alt
int16 seq
int8 inter_exist
